demos/background_subtraction_demo/python/README.md

```diff
@@ -87,7 +106,8 @@ Options:
   -h, --help            Show this help message and exit.
   -m MODEL, --model MODEL
                         Required. Path to an .xml file with a trained model or
-                        address of model inference service if using OVMS adapter.
+                        address of model inference service if using ovms
```

Comment:
Please, return the uppercase `OVMS` abbreviation

Reply:
Fixed.